src/components/ProgressBar/index.tsx

```diff
@@ -110,29 +179,14 @@ const ProgressBar: FunctionComponent<ProgressBarProps> = ({
                 </div>
             );
         }
-        return (
-            <Grid container spacing={3}>
-                <Grid item xs={value && displayValue ? 11 : 12}>
-                    <LinearProgress
-                        variant="determinate"
-                        value={value || 100}
-                        classes={{
-                            colorPrimary: classes.colorPrimary,
-                            barColorPrimary: classes.barColorPrimary,
-                        }}
-                    />
-                </Grid>
-                {displayValue && value && (
-                    <Grid item xs={1}>
-                        <Text>{value}%</Text>
-                    </Grid>
-                )}
-            </Grid>
-        );
+
+        const ProgressBarComponent = variant === 'linear' ? LinearProgressComponent : CircularProgressWithLabel;
+
+        return <ProgressBarComponent value={value || 100} displayValue={displayValue} />;
     };
     return (
         <>
-            <Heading variant="h3">{label || ''}</Heading>
+            {displayHeading && <Heading variant="h3">{label || ''}</Heading>}
```

Comment:
It is ok to just use label to determine whether to render heading or not. label && {}

Reply:
Updated.